pkgs/development/tools/schemacrawler-bin/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, jre
+}:
+
+stdenv.mkDerivation rec {
+  pname = "schemacrawler-bin";
+  version = "16.20.4";
+  src = fetchurl {
+    url = "https://github.com/schemacrawler/SchemaCrawler/releases/download/v${version}/schemacrawler-${version}-bin.zip";
+    sha256 = "sha256-YozWe3JMCG+NtGZlDp2zSIV7oWaKaep0zs5Lfr/CSWU=";
+  };
+
+  buildInputs = [ unzip ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r {config,lib} $out/
+    cat >$out/bin/schemacrawler <<EOF
+    #!/bin/sh
+    ${jre}/bin/java -cp $out/lib/*:$out/config schemacrawler.Main "\$@"
+    EOF
+    chmod +x $out/bin/schemacrawler
+  '';
+
+  preferLocalBuild = true;
+
+  meta = with lib; {
+    description = "Database schema discovery and comprehension tool";
+    homepage = "https://www.schemacrawler.com/";
+    license = licenses.gpl3;
```

```suggestion
    license = with licenses; [ epl10 gpl3Only lgpl3Only ];
```
see https://github.com/schemacrawler/SchemaCrawler/blob/8c11b547c468f1b015010c363ca60066b677545b/schemacrawler-api/src/main/java/schemacrawler/Version.java#L13-L15